studio/schemaTypes/show.ts

```diff
@@ -0,0 +1,115 @@
+import { defineType, defineField } from 'sanity';
+
+export default defineType({
+  name: 'show',
+  title: 'Show',
+  type: 'document',
+  fields: [
+    defineField({
+      name: 'date',
+      title: 'Date',
+      type: 'date',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: 'startTime',
+      title: 'Start Time',
+      type: 'string',
+      description: 'Start time in HH:MM format (24-hour)',
+      validation: (Rule) => Rule.required().regex(/^([01]?[0-9]|2[0-3]):[0-5][0-9]$/, {
+        name: 'time format',
+        invert: false,
+      }).error('Please enter a valid time in HH:MM format (e.g., 09:00 or 14:30)'),
+    }),
+    defineField({
+      name: 'endTime',
+      title: 'End Time',
+      type: 'string',
+      description: 'End time in HH:MM format (24-hour)',
+      validation: (Rule) => Rule.required().regex(/^([01]?[0-9]|2[0-3]):[0-5][0-9]$/, {
+        name: 'time format',
+        invert: false,
+      }).error('Please enter a valid time in HH:MM format (e.g., 09:00 or 14:30)'),
+    }),
+    defineField({
+      name: 'host',
+      title: 'Host',
+      type: 'string',
+      description: 'Name of the show host or DJ',
+      validation: (Rule) => Rule.required(),
+    }),
```

Comment:
A show can have a name, and it can also have a DJ reference. It must have at least one. 

For example, "The Nooner w/ Dave Lindquist", where "The Nooner" is the name, and "Dave Lindquist" is a reference to Dave's DJ record.

Or, if there's no name, then the page would just show the DJ's name:  "Josh T. Landow"

Or, if there's a name but no DJ, then it would just show the name: "Y-Not Session Shuffle"